CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+## [0.63.48](https://github.com/brave/brave-browser/releases/tag/v0.63.48)
+
+ - Added Brave Ads. ([#4209](https://github.com/brave/brave-browser/issues/4209))
```

Comment:
For people that don't really understand what those are, it might be better to clarify here. Maybe:

`- Added the ability to opt-in to user private Brave ads.`

Reply:
Agreed 👍